src/promptflow/promptflow/_cli/_utils.py

```diff
@@ -351,6 +354,35 @@ def is_format_exception():
     return False
 
 
+def exception_handler(func, activity_name, custom_dimensions=None):
```

Comment:
maybe change the name to reflect that it also handles temelemtry, how about:
cli_exception_and_temeletry_handler

Reply:
ok